code/iaas/logic/src/main/resources/META-INF/cattle/core-process/spring-core-processes-context.xml

```diff
@@ -126,7 +126,8 @@
 
     <!-- Load Balancer Target -->
     <process:process name="loadbalancertarget.create" resourceType="loadBalancerTarget" start="requested" transitioning="activating" done="active" />
-    <process:process name="loadbalancertarget.remove" resourceType="loadBalancerTarget" start="active,activating" transitioning="removing" done="removed" />
+    <process:process name="loadbalancertarget.remove" resourceType="loadBalancerTarget" start="active,activating,updating-active" transitioning="removing" done="removed" />
+    <process:process name="loadbalancertarget.update" resourceType="loadBalancerTarget" start="active,activating" transitioning="updating-active" done="active" />
```

Comment:
Yea, as this resource never goes through regular activate/deactivate cycle. It will be in activating state in a split of a second during the initial link creation, and I want to make sure that if 2 updates are called quickly one by one for the same service, they don't error out.